tests/pages/base.py

```diff
@@ -0,0 +1,151 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.common.action_chains import ActionChains
+
+from pypom import Page, Region
+
+
+class BasePage(Page):
+
+    URL_TEMPLATE = '/{locale}'
+
+    def __init__(self, selenium, base_url, locale='en-US', **url_kwargs):
+        super(BasePage, self).__init__(selenium, base_url, locale=locale, **url_kwargs)
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda s: self.seed_url in s.current_url)
+        el = self.find_element(By.TAG_NAME, 'html')
+        self.wait.until(lambda s: el.get_attribute('data-ffo-opensanslight'))
+        return self
+
+    @property
+    def header(self):
+        return self.Header(self)
+
+    @property
+    def footer(self):
+        return self.Footer(self)
+
+    class Header(Region):
+        report_content_form_url = 'https://bugzilla.mozilla.org/form.doc'
+        report_bug_form_url = 'https://bugzilla.mozilla.org/form.mdn'
+        # locators
+        _root_locator = (By.ID, 'main-header')
+        _menu_locator = (By.ID, 'nav-main-menu')
+        _platform_submenu_trigger_locator = (By.XPATH, 'id(\'nav-platform-submenu\')/..')
+        _platform_submenu_locator = (By.ID, 'nav-platform-submenu')
+        _platform_submenu_item_locator = (By.CSS_SELECTOR, '#nav-platform-submenu a')
+        _feedback_link_locator = (By.XPATH, 'id(\'nav-contact-submenu\')/../a')
+        _feedback_submenu_trigger_locator = (By.XPATH, 'id(\'nav-contact-submenu\')/..')
+        _feedback_submenu_locator = (By.ID, 'nav-contact-submenu')
+        _report_content_locator = (By.CSS_SELECTOR, 'a[href^="' + report_content_form_url + '"]')
+        _report_bug_locator = (By.CSS_SELECTOR, 'a[href^="' + report_bug_form_url + '"]')
+
+        # is displayed?
+        @property
+        def is_displayed(self):
+            return self.root.is_displayed()
+
+        # nav is displayed?
+        @property
+        def menu_is_displayed(self):
+            return self.find_element(*self._menu_locator).is_displayed()
+
+        def platform_submenu_displays(self):
```

Comment:
Change to a `@property`. Also, consider changing the name to `is_platform_submenu_displayed`.